app/mailers/postmaster_mailer.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class PostmasterMailer < ApplicationMailer
```

Comment:
do we need to specify a name for a class? Not sure we will have several mailers in barong.
So, by convention its just Mailer ?